components/layout/WebsiteLayout.tsx

```diff
@@ -0,0 +1,15 @@
+import { AppHeader } from "./AppHeader";
+import { Footer } from "./Footer";
+type WebsiteLayoutProps = {
+  children: React.ReactNode;
+  sections?: { name: string }[];
+};
+export const WebsiteLayout = ({ children, sections }: WebsiteLayoutProps) => {
+  return (
+    <body className="bg-backgroundDark pb-4 h-screen m-auto pt-24">
+      <AppHeader sections={sections} />
+      <main>{children}</main>
+      <Footer />
+    </body>
```

Comment:
This is causing a few warnings 
**Warning: Expected server HTML to contain a matching <body> in <div>.**
**Warning: validateDOMNesting(...): <body> cannot appear as a child of <div>.**